templates/explore/repo_search.tmpl

```diff
@@ -30,7 +30,7 @@
 	</div>
 </form>
 {{if .OnlyShowRelevant}}
-	<div class="ui blue attached message explore-relevancy-note">
+	<div class="ui message explore-relevancy-note">
```

Comment:
Why  the class is not removed since the `ui.explore-relevancy-note` was deleted  in _explore.less ?